app/Http/Requests/UpdateArtist.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateArtist extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'moniker' => 'required|max:255', // TODO: removing `required` from this causes tests to fail ?
+            'alt_moniker' => 'max:255',
+            'email' => 'email',
```

Comment:
Seems like this was added accidentally. Can we remove it?

Reply:
I never added that, that was what you had in `StoreArtist` and you was using `StoreArtist` for the `update()` method, so i simply made a new `UpdateArtist` and put the same rules in there ... 

but either way if the email is passed with an update request it should in  fact be validated as valid email no ? I think all possible fields need a minimal validation even if we are just setting a `min:` and/or `max:` ...

we should ***not*** remove the `email` validation *just* to please a test ...

Reply:
My bad, you're absolutely correct here. I forgot that I had added email as a public profile field. It doesn't exist on the legacy site, and I forgot that I had added it intentionally.